cinema/views.py

```diff
@@ -0,0 +1,39 @@
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data, many=True)

```

Comment:
The `many=True` argument in `MovieSerializer(data=request.data, many=True)` is incorrect for a POST request that creates a single movie. Remove `many=True` to correctly handle single object creation.